packages/docs/src/locales/ru/ru.json

```diff
@@ -677,7 +677,9 @@
         "focused": "Применяет `focus` стиль",
         "canBeFocused": "Если `false`, тогда компонент не может получить фокус, используя мышь или клавиатуру",
         "immediateValidation": "Устанавливает выполнение валидации при монтировании компонента",
-        "inputClass": "Передает строку с классами непосредственно для самого `input` (а не его обертки)"
+        "inputClass": "Передает строку с классами непосредственно для самого `input` (а не его обертки)",
+        "counter": "Отображает счетчик введенных символов",
+        "maxLength": "Максимальное количество символов"
```

Comment:
How this prop works without counter on?

Reply:
Well, it's needs to show the max-length number.